database/index.js

```diff
@@ -1,12 +1,18 @@
 const mysql = require('mysql');
+const seed = require('./dbSeed.js');
 
 const connection = mysql.createConnection({
-    host: 'localhost',
-    user: 'root',
-    password: '12345',
-    database: 'property'
+  host: 'localhost',
+  user: 'root',
+  password: '12345',
+  database: 'property',
+});
+
+connection.connect((err)=>{
```

Comment:
@pauly-van  Do you have a script for your seed file? As in if you ran the file separately with node it could reset and repopulate all the property data by itself? Sequelize has a handy sync() function that you can pass in a param to and invoke sync({force: true}) to wipe and re-generate all the data if desired.